src/connectors/price-ranges/connectPriceRanges.js

```diff
@@ -158,13 +158,15 @@ export default function connectPriceRanges(renderFn, unmountFn) {
       },
 
       init({ helper, instantSearchInstance }) {
-        this._refine = this._refine.bind(this, helper);
+        this.refine = (...opts) => {
+          this._refine(helper, ...opts);
```

Comment:
refine only has two arguments, what about not spreading?

Reply:
That's indeed preferable to avoid spreading if we know the number of arguments 👍